rhel/repositoryscanner.go

```diff
@@ -299,27 +381,16 @@ type repo struct {
 	CPEs []string `json:"cpes"`
 }
 
-func (m *mappingFile) Get(ctx context.Context, rs []string) ([]string, error) {
-	s := map[string]struct{}{}
-	for _, r := range rs {
-		cpes, ok := m.Data[r]
-		if !ok {
-			zlog.Debug(ctx).
-				Str("repository", r).
-				Msg("repository not present in a mapping file")
-			continue
-		}
-		for _, cpe := range cpes.CPEs {
-			s[cpe] = struct{}{}
-		}
+// GetOne takes a repoid and reports the CPEs and if the repoid was known
+// beforehand.
+func (m *mappingFile) GetOne(ctx context.Context, repoid string) (cpes []string, ok bool) {
```

Comment:
nit: the named returns aren't really used, so we can delete them

Reply:
I added the named return(s) to document that the returned strings will be CPEs.